pkg/prow/job.go

```diff
@@ -231,3 +209,31 @@ func (j *Job) CreateChangedCovHtml() error {
 	}
 	return err
 }
+
+func getFilesAndCovList(fullDiff bool, prComment github.PrComment, localP, baseP cover.CoverageList) (changedFiles []string, deltaCovList cover.DeltaCovList, err error) {
+	if !fullDiff {
+		// get github pull request changed files' name
+		var ghChangedFiles, err = prComment.GetPrChangedFiles()
+		if err != nil {
+			logrus.Errorf("Get pull request changed file failed.")
```

Comment:
format `err` with this message and return to upstream seems better

Reply:
fixed~